DQM/DTMonitorClient/src/DTOccupancyTest.cc

```diff
@@ -283,7 +300,7 @@ int DTOccupancyTest::runOccupancyTest(TH2F *histo, const DTChamberId& chId,
   bool failCells = false;
 
   // Check that the chamber has digis
-  if(histo->Integral() == 0) {
+  if (getIntegral(histo,chId) == 0) {
```

Comment:
I think you're right. Is it still in time tomorrow for the integration in 800, if I make the change and check it again?